testgrid/tgrun/pkg/runner/vmi/embed/runcmd.sh

```diff
@@ -202,6 +202,14 @@ function collect_debug_info_after_kurl() {
     echo "";
 }
 
+function store_join_command() {
```

Comment:
this function needs some improvement but it can go after we merge this pull request.